packages/dds/task-manager/src/interfaces.ts

```diff
@@ -184,4 +184,11 @@ export interface ITaskManager extends ISharedObject<ITaskManagerEvents> {
 	 * Check whether this client can currently volunteer for a task.
 	 */
 	canVolunteer(): boolean;
+
+	/**
+	 * Gets the list of currently queued task IDs.
+	 *
+	 * @remarks This method is optional for backwards compatibility. It will be required in the future.
```

Comment:
@jikim-msft If we add a method like this, we will have to take this approach of making it optional on main. But we can subsequently make it required on `next` (as soon as main has been merged into it). We don't expect external users to implement the interface themselves, so the `TaskManager` class should be the only implementation.